### PR TITLE
DOCS-879: Address Gantry API changes

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

**Comment by randhid** [comment]
Removing myself and adding Martha to review this pr.

File: docs/components/gantry/_index.md [hunk 1]
```diff
@@ -140,14 +141,15 @@ position, err := myGantry.Position(context.Background(), nil)
 
 ### MoveToPosition
 
-Move the axes of the gantry to the desired positions (mm).
+Move the axes of the gantry to the desired positions (mm) at the requested speeds.
```
Comment (martha-johnston) [inline docs/components/gantry/_index.md:144]: speeds are in mm per sec not sure if you want to add that unit as well

**Review by martha-johnston — APPROVED** [review]
just a couple of small things that you can feel free to change at your discretion. after that, LGTM!

File: docs/components/gantry/_index.md [hunk 2]
```diff
@@ -188,8 +193,58 @@ myGantry, err := gantry.FromRobot(robot, "my_gantry")
...
-myGantry.MoveToPosition(context.Background(), examplePositions, nil)
+myGantry.MoveToPosition(context.Background(), examplePositions, exampleSpeeds, nil)
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Home
+
+Run the homing sequence of the gantry to return to its starting positions and return true once completed.
```
[the remark this answers is not in the document]
Comment (npentrel) [reply docs/components/gantry/_index.md:207]: I honestly wasn't entirely sure what this command does - the description on the SDK docs is this or even less specific... Happy with what you are suggesting

```suggestion
Run the homing sequence of the gantry to re-calibrate the axes with respect to the limit switches.
```

**Pull request merged**